### PR TITLE
Logger#error is now a method. Added test for no servers case

### PR DESCRIPTION
**Pull request by twohlix**
So this fixes the 0 server case. now it will log an error to stderr

### CODE REVIEW

File: lib/elbas/logger.rb [hunk 1]
```diff
@@ -4,17 +4,35 @@ module Elbas
...
+
     def info(message)
       $stdout.puts [prefix, message, "\n"].join
     end
 
+    def error(message)
+      $stderr.puts [error_prefix, message, "\n"].join
+    end
+
     def cyan(text)
```
Comment (lserman) [inline lib/elbas/logger.rb]: I think `cyan`,  `red`, and `color_text` should all be private

File: lib/elbas/logger.rb [hunk 2]
```diff
@@ -4,17 +4,35 @@ module Elbas
   module Logger
     include Capistrano::Doctor::OutputHelpers
 
+    PREFIX_TEXT = '[ELBAS] '
```
Comment (lserman) [inline lib/elbas/logger.rb]:
```suggestion
    PREFIX_TEXT = '[ELBAS] '.freeze
```

File: lib/elbas/logger.rb (hunk 1 above)
Comment (twohlix) [reply lib/elbas/logger.rb]: okay, thats fine, was just following current pattern.

**Comment by twohlix** [comment]
Cool. Addressed feedback.

**Pull request merged**